dataflow/manual/content.tex

```diff
@@ -842,6 +837,35 @@ \subsection{Formal Definition of the Control-Flow Graph}
         \label{tab:nodes}
     \end{longtable}
 
+\autoref{tab:nodesWithException} shows all node types whish can possibly throw an exception and type to be thrown.
+Java class name of nodes are simplified as with \autoref{tab:nodes}.
+All of specific Exceptions type in table are in package \code{java.lang}.
```

Comment:
What do you mean with `specific`?
Why is `Exceptions` upper case?
I think the `of` can be removed, but some reformulation of that sentence would be good.